terraform/modules/tf_scheduled_queries/iam_roles.tf

```diff
@@ -0,0 +1,195 @@
+#
```

Comment:
please follow suggested conventions for comments in tf: https://www.terraform.io/docs/configuration/syntax.html#comments

Reply:
even for single-line comments, I think we tend to use `//`

Reply:
Fixed
https://github.com/airbnb/streamalert/pull/1128/commits/4d3aeb497e4a0a95e6a6648e0f7221a2fb06ad14